docs/book/architecture.rst

```diff
@@ -11,6 +11,42 @@ All bundles share the same conventions for naming things and the way of data per
 
 For deeper understanding of how Doctrine works, please refer to the `excellent documentation on their official website <http://doctrine-orm.readthedocs.org/en/latest/>`_.
 
+Fullstack Symfony
+-----------------
+
+.. image:: ../_images/symfonyfs.png
+    :scale: 15%
+    :align: center
+
+**Sylius** is based on Symfony2, which is an Open Source PHP Web applications development framework. Using Symfony allows
```

Comment:
"Open Source PHP Web applications development framework"

It's quite complicated explanation 🌿